geode-dunit/src/main/java/org/apache/geode/test/junit/rules/MemberStarterRule.java

```diff
@@ -229,10 +235,9 @@ public T withConnectionToLocator(int... locatorPorts) {
    */
   public T withJMXManager(boolean useProductDefaultPorts) {
     if (!useProductDefaultPorts) {
+      jmxPort = portSupplier.getAvailablePort();
```

Comment:
It looks like the old code would use a jmx manager port from the properties if one was set, while yours will always use a generated one.